Changelog

```diff
@@ -24,6 +24,38 @@ Gerhard (SG) and Eric Larson (EL).
 
 References like "pr/298" refer to github pull request numbers.
 
+Upcoming release
+================
+
+New features
+------------
+* ``nib-diff`` command line tool for comparing image files (pr/617)
+  (Chris Cheng, reviewed by Yaroslav Halchenko, Pradeep Raamana and CM)
```

Comment:
I love the appearance of my name, but you could add me and Chris with abbreviations (YOH, CC?) as "Other contributors" in the section above where you introduce them in general.  I think we will keep appearing here from time to time ;)